### PR TITLE
Updated appsettings.json

### PR DESCRIPTION
**Pull request by OlegoO**
Updated appsettings.json to simplify https://virtocommerce.com/docs/latest/vc-quickstart/. 
Allows preventing appsettings.json modifications to get a working solution.
Artifact URL: https://vc3prerelease.blob.core.windows.net/packages/VirtoCommerce.Platform.3.77.0-pr-2366.zip

### CODE REVIEW

**Comment by mvktsk** [comment]
Review task https://virtocommerce.atlassian.net/browse/VP-7696 has been created

**Review by ksavosteev — COMMENTED** [review]
What about https://github.com/VirtoCommerce/vc-platform/blob/dev/src/VirtoCommerce.Platform.Web/Properties/launchSettings.json? The default port is :10645 there.

**Review by akak1977 — APPROVED** [review]

**Review by ksavosteev — COMMENTED** [review]
Runs ok exept the IIS Express configuration. IIS Express configuration doesn't work out of the box with ssl port 5001, it needs a port between 44300 and 44399. See https://docs.microsoft.com/en-us/iis/extensions/using-iis-express/handling-url-binding-failures-in-iis-express#using-a-custom-ssl-port

**Review by Vectorfield4 — APPROVED** [review]

File: src/VirtoCommerce.Platform.Web/appsettings.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
         "Provider": "FileSystem",
         "FileSystem": {
             "RootPath": "~/assets",
-            "PublicUrl": "http://localhost:10645/assets/"
+            "PublicUrl": "https://localhost:5051/assets/"
```
Comment (Vectorfield4) [inline src/VirtoCommerce.Platform.Web/appsettings.json:74]:
```suggestion
            "PublicUrl": "https://localhost:5001/assets/"
```

**Comment by Vectorfield4** [comment]
> Runs ok exept the IIS Express configuration. IIS Express configuration doesn't work out of the box with ssl port 5001, it needs a port between 44300 and 44399. See https://docs.microsoft.com/en-us/iis/extensions/using-iis-express/handling-url-binding-failures-in-iis-express#using-a-custom-ssl-port

We should update this doc with info above https://github.com/VirtoCommerce/vc-platform/blob/master/docs/getting-started/deploy-from-precompiled-binaries-windows.md#host-on-windows-with-iis

**Review by Vectorfield4 — APPROVED** [review]

**Pull request merged**